### DISCUSSION TITLE
cursor styling not showing in the global css

### DISCUSSION THREAD

**Discussion by AlaaMX** [katspaugh/wavesurfer.js#3727] [Q&A] [answered]
Hello I'm trying to style the waveform cursor as shown in the example in the official docs 

      #waveform ::part(cursor) {
        height: 100px;
        top: 28px;
        border-radius: 4px;
        border: 1px solid #fff;
      }

      #waveform ::part(cursor):after {
        content: '🏄';
        font-size: 1.5em;
        position: absolute;
        left: 0;
        top: -28px;
        transform: translateX(-50%);
      }

im using nextjs and putting the styles in the global.css file but seems not working, thanks!

**Comment by katspaugh** [comment, marked as answer]
Did you create a container div with the id `wavesurfer`?

**Comment by AlaaMX** [comment]
Oh Okey I assumed that will be added in the initialization by the library.
